### PR TITLE
Add http status method to command exceptions

### PR DESCRIPTION
**Pull request by shadowhand**
Also changes the missing option code to be more agnostic.

### CODE REVIEW

**Comment by shadowhand** [comment]
@ameech CR please?

**Comment by ameech** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/equip/command/pull-request/3/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=equip/command)

**Comment by shadowhand** [comment]
@ameech fixed conflict, reup?

File: tests/CommandTest.php [hunk 1]
```diff
@@ -103,6 +99,12 @@ public function testRequiredOptionsFailure()
                 'user_id',
             ]);
 
+        $this->setExpectedException(
+            CommandException::class,
+            '',
+            CommandException::MISSING_OPTION
```
Comment (shadowhand) [inline tests/CommandTest.php:105]: @ameech I'll fix the concern with #5 here.

**Comment by ameech** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/equip/command/pull-request/3/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=equip/command)

**Pull request merged**